test/middleware.test.js

```diff
@@ -91,7 +91,7 @@ describe('loopback:middleware generator', function() {
         var newSources = Object.keys(
           readMiddlewaresJsonSync('server')['routes:after']);
         var expectedSources = builtinSources.concat(['my-middleware-4']);
-        expect(newSources).to.have.members(expectedSources);
+        expect(expectedSources).to.have.include.members(newSources);
```

Comment:
This looks rather weird to me. The argument of `expect()` is always the actual value as observed by the test, not the value we are expecting to see. Why is this change needed?